packages/twenty-front/src/modules/command-menu/hooks/useCommandMenu.ts

```diff
@@ -14,6 +15,9 @@ export const useCommandMenu = () => {
   const navigate = useNavigate();
   const setIsCommandMenuOpened = useSetRecoilState(isCommandMenuOpenedState);
   const setCommands = useSetRecoilState(commandMenuCommandsState);
+  const { resetSelectableListStates } = useSelectableList({
```

Comment:
I think we should not talk about "states" in the API of selectableList. It feels to me that states are an internal logic here.
What about `resetSelectedItem` instead?